php/Controllers/SettingsController.php

```diff
@@ -17,45 +17,198 @@
+     */
+    const settings_slug_advanced        = 'coyote_fields_advanced';
+
+    /**
+     * settings section slug for advanced fields
+     * used with add_settings_section()
+     * @var string advanced_settings_section
+     */
+    const advanced_settings_section     = 'advanced_settings_section';
+
+    /**
+     * WP Admin settings tools page title
+     * @var string subpage_title_tools
+     */
+    private $subpage_title_tools;
+
+    /**
+     * WP Admin settings main menu title
+     * @var string menu_title_main
+     */
+    private $submenu_title_tools;
+
+    /**
+     * WP Admin settings tools admin url slug admin.php?page=[slug]
+     * @var string submenu_tools_slug
+     */
+    const submenu_tools_slug            = 'coyote-tools';
+
+    /**
+     * Page slug used for tools settings
+     * used with settings_field(), do_settings_section() and register_setting()
+     * @var string settings_slug_tools
+     */
+    const settings_slug_tools           = 'coyote_fields_tools';
+
+    /**
+     * settings section slug for tools fields
+     * used with add_settings_section()
+     * @var string tools_settings_section
+     */
+    const tools_settings_section        = 'tools_settings_section';
+
+    /**
+     * @var mixed batch_job WordPress transient
+     */
     private $batch_job;
+
+    /**
+     * @var bool profile_fetch_failed
+     */
     private bool $profile_fetch_failed;
 
+    /**
+     * Constructor
+     */
     function __construct() {
-        $this->page_title = __('Coyote settings', COYOTE_I18N_NS);
-        $this->menu_title = __('Coyote', COYOTE_I18N_NS);
 
-        $this->profile_fetch_failed = false;
-        $this->profile = $this->getProfile();
+        /*
+         * Set page and menu titles via i18n functions
+         */
+        $this->page_title_main          = __('Coyote settings', COYOTE_I18N_NS);
+        $this->menu_title_main          = __('Coyote', COYOTE_I18N_NS);
+        $this->subpage_title_advanced   = __('Coyote advanced', COYOTE_I18N_NS);
+        $this->submenu_title_advanced   = __('Advanced', COYOTE_I18N_NS);
+        $this->subpage_title_tools      = __('Coyote tools', COYOTE_I18N_NS);
+        $this->submenu_title_tools      = __('Tools', COYOTE_I18N_NS);
 
-        $this->batch_job = BatchImportHelper::getBatchJob();
+        /*
+         * Set profile_fetch_failed to false and fetch profile
+         * when successful profile_fetch_failed will be set to true
+         */
+        $this->profile_fetch_failed     = false;
+        $this->profile                  = $this->getProfile();
 
-        $this->is_standalone = !!(get_option('coyote_is_standalone', false));
+        $this->batch_job                = BatchImportHelper::getBatchJob();
 
-        add_action('admin_enqueue_scripts', array($this, 'enqueue_scripts'));
+        /*
+         * Check if standalone mode is active
+         */
+        $this->is_standalone            = !!(get_option('coyote_is_standalone', false));
```

Comment:
I try to only use magic option strings in the `PluginConfiguration` class, can this data be fetched from there? I know it's not code you introduced, but it's good boyscouting while we're at it.

Reply:
Replaced the option retrieval with PluginConfiguration::isStandalone();